integration_test/buckets.tf.template

```diff
@@ -6,6 +6,25 @@ resource "aws_s3_bucket" "internal" {
     Application = "ORCA"
   }
 }
+# todo: Either commit to us-west-2 and remove user's ability to change it, or make 797873946194 variable with instructions.
+data "aws_iam_policy_document" "internal-policy-json" {
+ statement {
+   principals {
+     type        = "AWS"
+     identifiers = ["arn:aws:iam::797873946194:root"]
```

Comment:
See note above on removing accountid

Reply:
This is not a basic account ID. Added link.